app/assets/stylesheets/earthworks.css

```diff
@@ -76,4 +76,8 @@
 .documents-list .documentHeader .index_title {
   margin-left: .75rem;
   align-self: center;
+}
+
+.help-text.viewer_protocol {
+  display: none;
 }
```

Comment:
I know it isn't in the designs but I'm slightly worried about hiding this, if only because it makes it way easier in testing to see what viewer is actually getting rendered/what the data is supposed to be. Maybe we need to check if @dbranchini hid it intentionally, and if not if there's a way to re-add it to the page?

Reply:
I will remove it for now and we can add it back in if Darcy says it was purposeful

Reply:
I'm not sure what this is referring to? What did I eliminate? It probably wasn't intentional.